config/sagemaker/config.go

```diff
@@ -14,4 +14,10 @@ func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("aws_sagemaker_device_fleet", func(r *config.Resource) {
 		r.Path = "devicefleet"
 	})
+	p.AddResourceConfigurator("aws_sagemaker_endpoint", func(r *config.Resource) {
+		r.References["endpoint_config_name"] = config.Reference{
+			Type: "EndpointConfiguration",
```

Comment:
Could you please use the following configuration which is safer?

```suggestion
			TerraformName: "aws_sagemaker_endpoint_configuration",
```